### PR TITLE
Blog Post: Why Docker?

### PR DESCRIPTION
**Pull request by nzoschke**
screen shot 2016-06-29 at 9 27 26 am

## Release Playbook
- [x] Rebase against master
- [x] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by nzoschke** [comment]
Could use copy review and proof reading. Also the "Docker, Docker, Docker" image label needs a better style

**Comment by nzoschke** [comment]
Looks like the iframe for the gist isn't working yet either.

File: _posts/2016-06-29-why-docker.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+![Industry Cooperation](https://medium2.global.ssl.fastly.net/max/2328/1*UWOC6AEQXQUaLYMgSCheLg.png)*Industry Cooperation*
+
+## Why Not Docker?
+
+When talking about just the Image APIs I don’t see any reason why you wouldn’t target Docker Images when building something new.
+
+I ask this question to you… Is there a distribution format that’s simpler, easier or more supported than Docker Images now?
+
+That said, I do understand and share concerns about other parts Docker.
```
Comment (mwarkentin) [inline _posts/2016-06-29-why-docker.md]: `other parts Docker` -> `other parts of Docker`.

File: _posts/2016-06-29-why-docker.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+If you don’t know what you are doing, you can end up with a Docker system that is overly complex on the container, network, persistence and scheduling fronts.
+
+## Conclusion
+
+Why Docker? We have always needed tools to package, distribute and run software. Docker took a fresh approach to all this and is now widely supported across the entire industry.
+
+Building Docker Images and running software with the Docker Container APIs guarantees operability between every personal computer and CI and Cloud providers.
+
+Building AMIs, making Homebrew scripts, managing an APT repositories and and writing custom shell scripts and Makefiles didn’t come anywhere close.
```
Comment (mwarkentin) [inline _posts/2016-06-29-why-docker.md]: `managing an APT repositories` -> `managing an APT repository` OR `managing APT repositories`

File: _posts/2016-06-29-why-docker.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+I ask this question to you… Is there a distribution format that’s simpler, easier or more supported than Docker Images now?
+
+That said, I do understand and share concerns about other parts Docker. 
+
+For starters, if you’re happy with your current packaging and runtime solutions, don’t throw them away. Conservatism will pay off as the container space rapidly evolves. The hype cycle is definitely in effect.
+
+Next, I recommend caution in going deeper into the Docker universe than Images and Container APIs right now.
+
+There is interesting work in the Docker container scheduling space (Swarm, Mesos, Kuberenetes, Nomad) but these are extremely complex systems that can bring in massive headaches for you and your team. These dynamic systems make networking, load balancing, logging and debugging **harder than ever before**.
```
Comment (mwarkentin) [inline _posts/2016-06-29-why-docker.md]: `Kuberenetes` -> `Kubernetes`

**Comment by mattmanning** [comment]
Fixed the "Docker, Docker, Docker" image label by adding a caption class to the styles.

Couldn't get the iframe working but converted it to an inline code block.

**Comment by mattmanning** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;G1s7d7Nh&#x2F;123-why-docker"> why docker</a>

**Pull request merged**